src/lib/types.ts

```diff
@@ -328,6 +328,7 @@ export namespace Tz {
         state: KeyValue;
         endpoint_name: string | undefined;
         membersState?: {[s: string]: KeyValue};
+        publish?: Publish;
```

Comment:
```suggestion
        publish: Publish;
```

I think it should not be optional

Reply:
As it is used in other places I took the easy way out 😅

Reply:
I think it should be mandatory, would this give any problems?

Reply:
Re checking it is used in z2m availabilty

Only problem I see would be compatibility for other users of zh /zhc

Reply:
That is not a problem, next zhc release is a breaking one anyway